config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -134,3 +134,48 @@ periodics:
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
     testgrid-tab-name: capi-periodic-upgrade-main-1-25-1-26
+
+- name: periodic-cluster-api-provider-azure-e2e-workload-upgrade-1-26-1-27-main
```

Comment:
As part of this change I think we should also remove the 1.23 to 1.24 upgrade

Reply:
Good call, I'll make that change.